packages/dd-trace/src/span_pointers.js

```diff
@@ -0,0 +1,33 @@
+const crypto = require('crypto')
+
+const SPAN_LINK_KIND = 'span-pointer'
```

Comment:
this should probably be called `SPAN_LINK_SPAN_POINTER_KIND` or something. it makes sense as `SPAN_LINK_KIND`  in this file, but it becomes confusing when it's imported elsewhere.

Reply:
Yes I agree, good catch!